javagen/src/main/java/com/azure/autorest/model/clientmodel/ServiceClient.java

```diff
@@ -180,10 +183,19 @@ public final String getDefaultCredentialScopes() {
         return defaultCredentialScopes;
     }
 
-    public final boolean builderDisabled() {
+    public final boolean isBuilderDisabled() {
         return builderDisabled;
     }
 
+    /**
+     * Gets the package name for builder and wrapper classes.
+     *
+     * @return the package name for builder and wrapper classes.
+     */
+    public String getBuilderPackageName() {
+        return builderPackageName;
+    }
```

Comment:
I call it `builderPackageName` but actually it is used for both builder and wrapper client.

Welcome recommendation.